src/Illuminate/Database/QueryException.php

```diff
@@ -64,7 +73,39 @@ public function __construct($connectionName, $sql, array $bindings, Throwable $p
      */
     protected function formatMessage($connectionName, $sql, $bindings, Throwable $previous)
     {
-        return $previous->getMessage().' (Connection: '.$connectionName.', SQL: '.Str::replaceArray('?', $bindings, $sql).')';
+        $details = $this->formatConnectionDetails();
+
+        return $previous->getMessage().' (Connection: '.$connectionName.$details.', SQL: '.Str::replaceArray('?', $bindings, $sql).')';
```

Comment:
Why didn't you move `Connection: {connectionName}` inside `formatConnectionDetails()`?